### ISSUE TITLE
Empty value blocks on workspace should give feedback

### ISSUE THREAD

**Issue by natalioc** [RITAccess/blockly#71]
**Describe the bug**

Empty value blocks give no feedback to the user when navigated to, or when R is used.

**Expected behavior**

Screen reader should announce "empty text value block" or some such feedback when block is navigated to.
